src/styles/_header.scss

```diff
@@ -0,0 +1,62 @@
+@use 'variables';
+@use 'fonts';
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+
+  height: variables.$headerHeight;
```

Comment:
Ensure that `variables.$headerHeight` is defined in your 'variables' file. This variable should specify the height of the header for consistent styling.